server/src/main/java/pl/edu/agh/RaftServer.java

```diff
@@ -108,8 +129,7 @@ public State getState() {
                     return Optional.of(response);
                 }),
                 Case(instanceOf(AppendEntries.class), ae -> {
-                    // TODO: check content and perform proper actions
```

Comment:
Sorry for removing ;) I'll get it back ;) I didn't implement it as it's connected to election ;) not log replication